core-container/entry.sh

```diff
@@ -300,18 +294,46 @@ fi
 if [ "$ESPW_NAME" == "" ]; then
     ESPW_NAME="espw"
 fi
-if [ "$Namespace" == "" ]; then
+if [ "$NAMESPACE" == "" ]; then
     NAMESPACE="default"
 fi
+if [ "$PROVIDER_NAME" == "" ]; then
+    PROVIDER_NAME="default"
+    export PROVIDER_NAME
+fi
+if [ "$PROVIDER_NAMESPACE" == "" ]; then
+    PROVIDER_NAMESPACE=spaceprovider-${PROVIDER_NAME}
+    export PROVIDER_NAMESPACE
+fi
+if [ "$PROVIDER_SECRET_NAME" == "" ]; then
+    PROVIDER_SECRET_NAME=psecret
+fi
+if [ "$PROVIDER_SECRET_NAMESPACE" == "" ]; then
+    PROVIDER_SECRET_NAMESPACE=psecret_namespace
+fi
+if [ "$SPACE_MANAGER_KUBECONFIG" == "" ]; then
+    # if the space_manager_kubeconfig is not set, then we assume the kubestellar
+    # cluster is the space manager cluster.
```

Comment:
I use the term "hosting cluster" for the cluster into which the multi-cluster configuration management core processes are deployed as workload. Andy likes "core" cluster.